sql/codegen_xgboost_test.go

```diff
@@ -32,6 +32,11 @@ COLUMN sepal_length, sepal_width, petal_length, petal_width
 LABEL class 
 INTO sqlflow_models.my_xgboost_model;
 `
+const testAnalyzeTreeModelSelectIris = `
+SELECT * FROM iris.train
+ANALYZE sqlflow_models.my_xgboost_model
```

Comment:
sqlflow_models.my_xgboost_model -> XGB.my_xgboost_model ?

https://github.com/Yancey1989/sqlflow/blob/497ca9691a006652a4a7b6b0e8e46267d69543c4/sql/codegen_analyze.go#L88-L90

Reply:
`XGB.` is the prefix of Estimator, I think the model name can be any user-defined name.

Reply:
Sorry, I misunderstood the concepts of estimator and model.